src/modules/core/components/ActionsList/ActionsListItem.tsx

```diff
@@ -0,0 +1,207 @@
+   * title?: string | messageDescriptor,
+   * topic?: string
+   * createdAt: Date,
+   * domain?: DomainType,
+   * commentCount?: number,
+   * status?: number
+   */
+  item: any;
+  handleOnClick?: (handlerProps: ClickHandlerProps) => void;
+}
+
+const ActionsListItem = ({
+  item: { id, userAddress, status, topic, domainId },
+  item,
+  handleOnClick = () => undefined,
+}: Props) => {
+  const { formatMessage, formatNumber } = useIntl();
+
+  const popoverPlacement = useMemo(() => {
+    const offsetSkid = (-1 * removeValueUnits(popoverWidth)) / 2;
+    return [offsetSkid, removeValueUnits(popoverDistance)];
+  }, []);
+
+  const handleSyntheticEvent = useCallback(
+    () => handleOnClick({ id, userAddress, topic, domainId }),
+    [handleOnClick, id, userAddress, topic, domainId],
+  );
+
+  const { Decorate } = new TextDecorator({
+    username: (usernameWithAtSign) => (
+      <UserMention
+        username={usernameWithAtSign.slice(1)}
+        showInfo
+        popperProps={{
+          showArrow: false,
+          placement: 'bottom',
+          modifiers: [
+            {
+              name: 'offset',
+              options: {
+                offset: popoverPlacement,
+              },
+            },
+          ],
+        }}
+      />
+    ),
+  });
+
+  return (
+    <li>
+      <div
+        /*
+         * @NOTE This is non-interactive element to appease the DOM Nesting Validator
+         *
+         * We're using a lot of nested components here, which themselves render interactive
+         * elements.
+         * So if this were say... a button, it would try to render a button under a button
+         * and the validator would just yell at us some more.
+         *
+         * The other way to solve this, would be to make this list a table, and have the
+         * click handler on the table row.
+         * That isn't a option for us since I couldn't make the text ellipsis
+         * behave nicely (ie: work) while using our core <Table /> components
+         */
+        role="button"
+        tabIndex={0}
+        className={getMainClasses({}, styles, { [Status[status]]: !!status })}
+        onClick={handleSyntheticEvent}
+        onKeyPress={handleSyntheticEvent}
+      >
+        <div className={styles.avatar}>
+          {userAddress && (
+            <UserAvatar
+              size="s"
+              address={userAddress}
+              notSet={false}
+              showInfo
+              popperProps={{
+                showArrow: false,
+                placement: bottom,
+                modifiers: [
+                  {
+                    name: 'offset',
+                    options: {
+                      offset: popoverPlacement,
+                    },
+                  },
+                ],
+              }}
+            />
+          )}
+        </div>
+        <div className={styles.content}>
+          <div className={styles.title} title={item.title}>
+            <Decorate>{item.title}</Decorate>
```

Comment:
I would make sure that item.title exist - if title is null, then app is blowing

Reply:
Nice catch! I'll make the change.

Also:
> app is blowing

:rofl: